### ISSUE TITLE
Update ? Translate ?

### ISSUE THREAD

**Issue by SebCanet** [gasolin/BlocklyDuino#14]
Hello,
I tested your project and it works fine ! But I have a few questions...
I would like to know if you think to update the blockly code inside blocklyduino ?
I had to look to translate, how could I help to make it internationalized ?

Thanks a lot.

**Comment by SebCanet** [comment]
We're different teacher, and worskshop animator from different part in France, and we will start to work on the fork I did.
But what we will create could be useful to other kids in other countries with another teachers....

But I think we'll have to change directories of different file, would you like to help us ?
I started to clean up to keep only arduino generator.

Thanks.

**Comment by SebCanet** [comment]
Well, once again I'll ask you about internationalize, because a spanish teacher asked to translate my fork. However I prefer to have only one repository if possible.
And as you wrote name in 'hard' in blocks, it won't be possible...
Is it possible instead to use the method 
var name = Blockly.Procedures.findLegalName(     Blockly.Msg.PROCEDURES_DEFNORETURN_PROCEDURE, this);

Thanks a lot.

**Comment by gasolin** [comment]
In newly update version I think you could follow Blockly guide to internationalize BlocklyDuino.

**Comment by SebCanet** [comment]
Yes, but you have to explain how to use text from database in js file '\msg\js'
Because if new block are with text instead of reference to text, we cannot internationalize.

**Comment by SebCanet** [comment]
I tried to explain how to create multilingual block here : 
http://tic.technologiescollege.fr/wiki/doku.php/en/blockly_rduino/create_blocks
I hope it's understandable...tell what I can enhance.

**Comment by bolanka** [comment]
Hi Sebcanet. I need same thing for my country. Your link broken. Please update. Thanks

**Comment by SebCanet** [comment]
Sorry, new link:
[http://www.libreduc.cc/wiki/doku.php/en/blockly_rduino/create_blocks](http://www.libreduc.cc/wiki/doku.php/en/blockly_rduino/create_blocks)
Tell me if you need more information.

**Comment by bolanka** [comment]
Thanks SebCanet
